### PR TITLE
BMCP-10 : Footer component

### PR DESCRIPTION
**Pull request by nikhilsharma0025**
## Description
Added footer component and also the iframe

#### Added 
- Footer.tsx file 
- Footer.css

### CODE REVIEW

**Comment by nikhilsharma0025** [comment]
@nik-1236789  the routes.tsx issue you are talking about , So in my previous pr i have deletd it and this will work fine now.

File: src/components/footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+}
+.row{
+    display: flex;
+    flex-wrap: wrap;
+}
+ul{
+    list-style: none;
+}
+.footer{
+    background: linear-gradient( to bottom,#68EACC 0% , #497BE8 100%) !important;
```
Comment (2SkeePeR2) [inline src/components/footer/Footer.css:18]: A good thing to know -> Avoid using ```!important``` wherever possible.

File: src/components/footer/Footer.tsx [hunk 2]
```diff
@@ -1,12 +1,57 @@
...
+        <footer className="footer">
+            <div className="container">
+                <div className="row">
+                    <div className="footer-col">
+                        <h4>Discover</h4>
+                        <ul>
+                            <li><a href="#"> Home</a></li>
+                            <li><a href="#"> Sales</a></li>
+                            <li><a href="#">Our Network </a></li>
+                            <li><a href="#">Office Automation </a></li>
```
Comment (2SkeePeR2) [inline src/components/footer/Footer.tsx:16]: Are you not gonna link the pages ?

File: src/components/footer/Footer.tsx [hunk 3]
```diff
@@ -1,12 +1,57 @@
...
+                    </div>
+                </div>
+                <div className='row'>
+                    <p className='copyright'>
+                        BMCP Solutions India Pvt. Ltd., Φ73 Patpargunj Industrial Area New Delhi 110092
+                    </p>
+                </div>
+                <div className='row1'>
+                    <p className='copyright1'>
+                        Copyright ©  ' {new Date().getFullYear()} BMCP Inc. All rights reserved.
```
Comment (2SkeePeR2) [inline src/components/footer/Footer.tsx:50]: This is not how you show copyright symbols on website/HTML. Refer to [this](https://www.w3schools.com/html/html_symbols.asp) for the correct method.

Comment (2SkeePeR2) [inline src/components/footer/Footer.tsx:50]: This is not how you show copyright symbols on website/HTML. Refer to [this](https://www.w3schools.com/html/html_symbols.asp) for the correct method.

**Review by 2SkeePeR2 — CHANGES_REQUESTED** [review]

File: src/components/footer/Footer.tsx (hunk 2 above)
Comment (nikhilsharma0025) [reply src/components/footer/Footer.tsx:16]: the pages are are not created yet

**Pull request merged**